src/config/couchdb.rst

```diff
@@ -194,3 +194,16 @@ Base CouchDB Options
            parameter was defined: :config:option:`httpd/max_http_request_size`,
            which can be used to limit maximum http request sizes. After upgrade,
            it is advisable to review those settings and adjust them accordingly.
+
+    .. config:option:: enable_database_recovery :: Enable database recovery
+
+        Enable this to only "soft-delete" databases when
+        :ref:`DELETE /{db} <api/db>` DELETE  requests are made. This will place
+        a ``.recovery`` directory in your data directory and move deleted
+        databases/shards there instead. You can then manually delete these
+        files later, as desired.
+
```

Comment:
The same for full text index, i.e.  index generated by dreyfys/clouseau. The directory containing full text index will be renamed from `foo.1234567890` to `foo.20170912.092828.deleted.1234567890` for recovery.

Reply:
@jiangphcn just to clarify does `enable_database_recovery` toggles index soft deletion as well or are they configured independently? Do views follow the same behavior?

Reply:
What happens in the case of deleting multiple databases with the same name?  i.e. create "foo", soft-delete "foo", re-create "foo", soft-delete "foo".

Is the original "foo" lost, or are these disambiguated in some way?

Reply:
@willholley one setting for all, at least for full text index https://github.com/apache/couchdb/blob/0155bd884bd8c455cc4057811cf67035b04c427f/src/dreyfus/src/dreyfus_index_manager.erl#L122. Views follow the same behavior.

Reply:
> What happens in the case of deleting multiple databases with the same name? i.e. create "foo", soft-delete "foo", re-create "foo", soft-delete "foo".
good question and I don't know - I just copied the description from default.ini

Reply:
there will be timestamp to distinguish them with the same database name, such as `foo.20170912.092828.deleted.1234567890`.